charts/nri-kube-events/templates/_helpers.tpl

```diff
@@ -1,154 +1 @@
 {{/* vim: set filetype=mustache: */}}
```

Comment:
same here, any reason to keep the file ?

Reply:
Updated in 8488d1d0d4b77194599e360989414f6fc7778b3d.